app/views/appointments/index.html.erb

```diff
@@ -1 +1,4 @@
 <h1>appointments</h1>
+<%= @pet.appointments.each do |f| %>
```

Comment:
what is this?
Doesnt pushing this overwrite all my code?

Reply:
the github is displaying very strangely... if this problem is completly fixed then ill push it... but i cant tell lol

Reply:
That's because this was my first push before I pulled your index.

Reply:
@noliver-ykc please review the latest commit on this pr

Reply:
It shouldn't be deleted. That my code from before I pulled yours, also it's the index is on my local host so there's no way it was deleted.